lib/policy.py

```diff
@@ -1612,7 +1630,7 @@ def __ne__(self, other):
     'timeout': 'TIMEOUT',
     'traffic-type': 'TRAFFIC_TYPE',
     'verbatim': 'VERBATIM',
-    'vpn': 'VPN',
+#    'vpn': 'VPN',
```

Comment:
Indeed no. This is a workaround specific to our setup, as vpn is a zone name and not a special keyword.
Cf. https://github.com/google/capirca/issues/62
I removed the comment sign.